src/main/resources/application.yaml

```diff
@@ -67,6 +67,9 @@ service-config:
       case-type-ids:
         - Bulk_Scanned
       allow-creating-case-before-payments-are-processed: true
+      form-type-to-surname-ocr-field-mappings:
+        - formType: B123
+          ocrField: field_surname
```

Comment:
just a note for QA I guess - need to be aware that changes will be visible in ccd UI for B123 form which has `field_surname` inside OCR data 🙂

Reply:
in future this will be extending to multiple fields/forms. So probably we can have jurisdiction at top and list of forms and then each form will have list of fields like surname first name etc

Reply:
this config was there , lets keep it as it is, latter depending on requirements  we can improve it